ui/src/app/tasks/task-executions/task-executions.component.html

```diff
@@ -1,17 +1,17 @@
 <app-tasks-tabulation id="executions-list" *ngIf="taskExecutions" #tasksTabulation>
-
   <div dataflowLayoutType type="full">
-    <div class="list-bar" *ngIf="!(taskExecutions?.totalPages < 2 && taskExecutions.items.length === 0)">
-      <button class="btn btn-default btn-fa" (click)="refresh()" title="Refresh" type="button">
-        <span class="fa fa-refresh"></span>
-        Refresh
-      </button>
-    </div>
-
+    <app-list-bar [params]="params" [page]="taskExecutions" [countSelected]="countSelected()" #listBar
+                  (refresh)="refresh()" [hideSearch]="true" [actions]="executionsActions()"
+                  (action)="applyAction($event.action)">
+    </app-list-bar>
     <table *ngIf="taskExecutions?.items && taskExecutions.items.length > 0"
            class="table table-hover table-actions" id="taskExecutionsTable">
       <thead>
       <tr>
+        <th style="width: 30px" [dataflowAppRoles]="['ROLE_DESTROY']">
+          <app-master-checkbox (change)="changeCheckboxes()" *ngIf="form?.checkboxes" [dataflowAppRoles]="['ROLE_CREATE']"
+                               [items]="form.checkboxes"></app-master-checkbox>
+        </th>
         <th style="width: 140px">
           <app-sort id="sort-id" (change)="applySort($event)" [value]="'TASK_EXECUTION_ID'" [sort]="params">
             Execution Id
```

Comment:
Should call it `Task Execution Id` as there are also Job Execution ids etc, we should be more specific.